INSTALL.md

```diff
@@ -0,0 +1,71 @@
+### 1. Back-end setup
+
+Follow the [Sheer installation instructions](https://github.com/cfpb/sheer#installation)
+to get Sheer installed.
+**NOTE:** We suggest creating a virtualenv variable specific to this project,
+such as `cfgov-refresh` instead of `sheer` used in the Sheer installation instructions:
+
+```bash
+$ mkvirtualenv cfgov-refresh
+```
+
+Install the following dependencies into your virtual environment.
+We called ours `cfgov-refresh` (see previous step above):
+
+```bash
+$ workon cfgov-refresh
+
+$ pip install git+git://github.com/dpford/flask-govdelivery
+$ pip install git+git://github.com/rosskarchner/govdelivery
+```
+
```

Comment:
Moved config to subsection in back-end setup in INSTALL.md https://github.com/cfpb/cfgov-refresh/commit/597d7e1c26a6c71da369d9ad1bd9a44796a4673d